internal/rbd/controllerserver.go

```diff
@@ -772,7 +773,11 @@ func (cs *ControllerServer) createBackingImage(
 		}
 	}
 
-	log.DebugLog(ctx, "created image %s backed for request name %s", rbdVol, rbdVol.RequestName)
+	requestName, err := rbdVol.GetRequestName(ctx)
+	if err != nil {
+		requestName = "<unset request name>"
```

Comment:
@nixpanic why is this special case not an error case?

Reply:
It is only used for logging, and was not an error before this change.